packages/lesswrong/components/ea-forum/EABestOfPage.tsx

```diff
@@ -0,0 +1,600 @@
+  "8qXrou57tMGz8cWCL", // Are education interventions as cost effective as the top health interventions?
+];
+const featuredAudioPostIds = [
+  "jk7A3NMdbxp65kcJJ", // 500 million, but not a single one more
+  "rXYW9GPsmwZYu3doX", // What happens on the average day?
+  "ffmbLCzJctLac3rDu", // StrongMinds should not be a top rated charity (yet)
+];
+
+// TODO do useMulti's with these to speed things up
+const allPostIds = [
+  ...bestOfYearPostIds,
+  ...popularThisMonthPostIds,
+  ...featuredAudioPostIds,
+];
+
+const allSequenceIds = [
+  ...featuredCollectionsSequenceIds,
+  ...learnAboutEASequenceIds,
+  ...introToCauseAreasSequenceIds,
+];
+
+const allCollectionIds = [
+  ...learnAboutEACollectionIds,
+];
+
+const PostListItem = ({
+  documentId,
+  isNarrow = false,
+  classes,
+}: {
+  documentId: string;
+  isNarrow?: boolean;
+  classes: ClassesType;
+}) => {
+  const { Loading, TruncatedAuthorsList, ForumIcon } = Components;
+  const authorExpandContainer = useRef(null);
+
+  const { document, loading } = useSingle({
+    documentId,
+    collectionName: "Posts",
+    fragmentName: "PostsPage",
+  });
+
+  const {eventHandlers} = useHover({pageElementContext: "postListItem",  documentId: documentId, documentSlug: document?.slug})
+
+  const postLink = document ? postGetPageUrl(document) : "";
+
+  if (loading) return <Loading />;
+
+  if (!document) return null;
+
+  const timeFromNow = moment(new Date(document.postedAt)).fromNow();
+  const ago = timeFromNow !== "now" ? <span className={classes.xsHide}>&nbsp;ago</span> : null;
+
+  return (
+    <div {...eventHandlers} className={classes.postListItem}>
+      <div className={classes.postListItemTextSection}>
+        <div className={classes.postListItemTitle}>
+          <Link to={postLink}>{document.title}</Link>
+        </div>
+        <div className={classes.postListItemMeta}>
+          <div ref={authorExpandContainer}>
+            <InteractionWrapper>
+              <TruncatedAuthorsList post={document} expandContainer={authorExpandContainer} />
+            </InteractionWrapper>
+          </div>
+          &nbsp;·&nbsp;
+          {timeFromNow}
+          {ago}
+          &nbsp;·&nbsp;
+          {document.readTimeMinutes}m read
+          <div>
+            {!isNarrow && (
+              <span className={classNames(classes.commentCount, classes.xsHide)}>
+                &nbsp;·&nbsp;
+                <Link to={`${postLink}#comments`} className={classes.commentCount}>
+                  <ForumIcon icon="Comment" />
+                  {document.commentCount}
+                </Link>
+              </span>
+            )}
+          </div>
+        </div>
+        <div className={classes.postListItemPreview}>{document.contents?.plaintextDescription}</div>
+      </div>
+      <img className={classes.postListItemImage} src={document.socialPreviewData.imageUrl || siteImageSetting.get()} />
+    </div>
+  );
+};
+
+const SequenceOrCollectionCard = ({
```

Comment:
It's quite similar to the event card, or the SequencesGridItem. The main reason I didn't reuse things was just to simplify the MVP to get it done faster, and keep everything in one file for tweaking purposee. E.g. adding the post count to the other sequences card would have required fetching the data in a more efficient way (probably adding a denormalised field), because this version just fetches all the posts which is too slow for the polished version imo